CompilerSource/compiler/components/write_object_data.cpp

```diff
@@ -18,24 +18,22 @@
 **/
 
 #include "makedir.h"
-#include <stdio.h>
-#include <iostream>
-#include <fstream>
-#include <algorithm>
-
-using namespace std;
-
-
 #include "parser/parser.h"
-
 #include "backend/GameData.h"
 #include "compiler/compile_common.h"
 #include "event_reader/event_parser.h"
 #include "general/parse_basics_old.h"
 #include "settings.h"
-
 #include "languages/lang_CPP.h"
 
+#include <stdio.h>
+#include <iostream>
+#include <fstream>
+#include <algorithm>
+#include <vector>
```

Comment:
Same comment I made on #1608, you aren't using vector at all, stop including it. You would want the generated header to include vector.

Reply:
Nevermind, if you look, Josh was already using vector in this source file and the include for it was coming from one of the headers. Just leave the include then, I guess it's fine.